### PR TITLE
[13.0][ADD] sale_payment_sheet_financial_risk: New module

### PR DESCRIPTION
**Pull request by carlosdauden**
Extends Partner Financial Risk to allows you to deduct from the risk the amounts
collected on sale payment sheets pending reconciliation.

@Tecnativa TT37610

### CODE REVIEW

File: sale_payment_sheet_financial_risk/i18n/es.po [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+msgid "Set 0 if it is not locked"
+msgstr "Establece 0 si no está bloqueado"
+
+#. module: sale_payment_sheet_financial_risk
+#: model:ir.model.fields,field_description:sale_payment_sheet_financial_risk.field_res_partner__risk_sale_payment_sheet
+#: model:ir.model.fields,field_description:sale_payment_sheet_financial_risk.field_res_users__risk_sale_payment_sheet
+msgid "Total Sale Payment Sheet"
+msgstr "Total hoja de pagos de ventas"
+
+#~ msgid "Full risk computation"
```
Comment (sergio-teruel) [inline sale_payment_sheet_financial_risk/i18n/es.po]: Remove this

File: sale_payment_sheet_financial_risk/views/res_partner_view.xml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+                >
+                    <field
+                        name="risk_sale_payment_sheet"
+                        nolabel="1"
+                        widget='monetary'
+                        options="{'currency_field': 'risk_currency_id'}"
+                    />
+                </button>
+            </field>
+<!--            <field name="risk_invoice_draft_limit" position="before">-->
```
Comment (sergio-teruel) [inline sale_payment_sheet_financial_risk/views/res_partner_view.xml]: Do you need this?

**Review by sergio-teruel — APPROVED** [review]
Little comments

**Comment by carlosdauden** [comment]
Changes done

File: sale_payment_sheet_financial_risk/readme/USAGE.rst [hunk 3]
```diff
@@ -0,0 +1,4 @@
+To use this module, you need to:
+
+#. Go to *Customers > Financial Risk*
+#. Choose options to compute in credit limit.
```
Comment (pedrobaeza) [inline sale_payment_sheet_financial_risk/readme/USAGE.rst]: Which options? Please complete this

Comment (carlosdauden) [reply sale_payment_sheet_financial_risk/readme/USAGE.rst]: Go to *Customers > Financial Risk*
and you can see all options to set...

Comment (pedrobaeza) [reply sale_payment_sheet_financial_risk/readme/USAGE.rst]: Yes, but write a bit more about it. Specifically, about the check that enables to check this amount, and the other limit. Also a bit about the sign.

Comment (carlosdauden) [reply sale_payment_sheet_financial_risk/readme/USAGE.rst]: Changes done

**Pull request merged**